src/cache.jl

```diff
@@ -333,6 +335,9 @@ MemoryLayout(C::Type{CachedArray{T,N,DAT,ARR}}) where {T,N,DAT,ARR} = cachedlayo
 
 MemoryLayout(::Type{<:AbstractCachedArray}) = GenericCachedLayout()
 
+transposelayout(::GenericCachedLayout) = GenericCachedLayout()
```

Comment:
Should this be:
```suggestion
transposelayout(::AbstractCachedLayout) = GenericCachedLayout()
```
(also next line)